src/services/db/index.ts

```diff
@@ -221,7 +221,7 @@ export default class DBService extends GenericService {
     await this.connections?.model.sync({ force: !!process.env.FORCE });
     await this.profiles?.model.sync({ force: !!process.env.FORCE });
     await this.posts?.model.sync({ force: !!process.env.FORCE });
-    await this.posts.vectorizeContent();
+    await this.posts.vectorizeContent().catch(e => console.error(e));
```

Comment:
:+1: 
Btw could you test the search in the UI since we implemented tsvector search?